test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
```

Comment:
Can we abstract the AJAX request into a `loadFile(fileName)` function that accepts a NodeFunction (a callback that returns an error in the first argument and the content in the second)?

```javascript
loadFile('../src/js.cookie.js', function loadedFile(error, libraryContent) {
  if (error) { throw error; }
  // rest of the code here
});
```